### ISSUE TITLE
Top10 Test Performance chart showing TestData 

### ISSUE THREAD

**Issue by vgopireddy-incomm** [adiralashiva8/testng-metrics#20]
![image](https://user-images.githubusercontent.com/46935946/53623731-ebda8780-3c23-11e9-97e5-11baf75b0849.png)

It would be nice if we display only Test case name any way we are showing Test data in Test metrics.

**Comment by adiralashiva8** [comment]
Agree, Will add new column to display test data (parameters). But I'm worried when same test case is executed multiple time using data provider then as a user I dont know which test case took more time
